insee/main.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Dec  4 09:38:28 2015
+
+"""
+
+from globals import path_data
+
+print("Initialisation...")
+from equipements import info_equipement
+from revenus import info_revenus
+from census import info_population
+
+
+equipement = info_equipement()
```

Comment:
yes.
Sor far 2013 and 2014 are implemented. I didn't check what the link was for years before 2013 on the insee website